x-team/xp-ui
# Add sticky sidebar to JobsPageLayout

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3495

## Description

Add sticky sidebar to JobsPageLayout

## Checklist


## Steps to Test or Reproduce

1. Go to http://localhost:9001/?path=/story/screens-and-layouts-jobspagelayout-use-cases--job-details-page
1. Resize the page from wider to narrow and observe how elements adapt responsively
1. Also observe how the JobsApplicantCard stick to the bottom of the viewport
1. Scroll until the bottom of the page and observe the ProTipCard bellow the content

## Impacted Areas in Application

- Mostly `JobsPageLayout`

## Screenshots

![2020-01-14 11 49 14](https://user-images.githubusercontent.com/131859/72354173-098dcf80-36c4-11ea-8c36-df80658381e3.gif)
